Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -107,75 +107,227 @@ For the first fetch, the integration will fetch incidents that were created 10 m
-2. Under the Incident Type dropdown, select **ServiceNow Ticket**.
-3. Change the mapping according to your needs.
-4. Save your changes.
-    
-##### 5 fields have been added to support the mirroring feature:
-- **dbotMirrorDirection** - determines whether mirroring is incoming, outgoing, or both. Default is Both.
-    - You can choose the mirror direction when configuring the ServiceNow instance using the **Incident Mirroring Direction** field.
-
-- **dbotMirrorId** - determines the incident ID in the 3rd party integration. In this case, the ServiceNow sys ID field.
-- **dbotMirrorInstance** - determines the ServiceNow instance with which to mirror.
-- **dbotMirrorLastSync** - determines the field by which to indicate the last time that the systems synchronized.
-- **dbotMirrorTags** - determines the tags that you need to add in Cortex XSOAR for entries to be pushed to ServiceNow.
-    - You can set the tags in the instance configuration, using **Comment Entry Tag**, **Work Note Entry Tag** and **File Entry Tag**.
-
-![image](https://raw.githubusercontent.com/demisto/content/d9bd0725e4bce1d68b949e66dcdd8f42931b1a88/Packs/ServiceNow/Integrations/ServiceNowv2/doc_files/mirror-fields.png)
-
-#### STEP 2 - Modify the outgoing mapper.
-1. Under **Classification and Mapping**, click **ServiceNow - Outgoing Mapper.**
-The left side of the screen shows the ServiceNow fields to which to map and the right side of the
+You can also attach files from either of the systems which will then be available in the other system.  
+
+The following instructions include steps for configuring the integration and incoming and outgoing mappers. However, they do not cover every option available in the integration nor classification and mapping features. 
+For information about classification and mapping see [Classification and Mapping](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-6/cortex-xsoar-admin/incidents/classification-and-mapping.html).  
+
+**Note:**  
+- For Cortex XSOAR version 6.1 only, the final source of truth for an incident are the values in Cortex XSOAR.  For example, if you change the severity in Cortex XSOAR and then change it back in ServiceNow, the final value that will be presented is the one in Cortex XSOAR. For versions 6.2 and later, if mirroring is in both directions then the latest update is the source of truth. 
+- The mirroring settings apply only for incidents that are fetched after applying the settings. Pre-existing comments or work notes are not fetched/mirrored at the time of incident creation.
+-  Confirm whether you will use ServiceNow out of the box or the Security Incident Response (SIR) module, since mirroring is not supported out of the box for SIR. 
+- To use a custom mapper, you must first duplicate the mapper and edit the field in the copy of the mapper. This detaches the instance, so the pack does not automatically get updates.
```

Comment:
If you duplicate the mapper, it not detaches the pack, only if you detach the OOB mapper and make changes on it.
For the copy, it will be like any custom content item